astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx

```diff
@@ -0,0 +1,86 @@
+
+**Jens:** Our clients mainly consist of financial institutions like banks, brokers and asset managers. But we also have partners in the Fintech industry.
+
+**Dan:** What is the unique problem Tangany solves?
+
+**Jens:** Tangany addresses the complexity and regulatory hurdles financial institutions face when entering the digital asset market. By offering a fully integrated, cloud-based custody solution that simplifies technical infrastructure challenges and ensures compliance with stringent regulatory requirements, Tangany provides a faster and more cost-effective solution for entering the digital asset market.
+
+> We were looking for a reliable authentication service to scale our solution, provide new features, and add additional security mechanisms to the existing system.
+
+**Dan:** What do you find to be the greatest challenge about the service you provide (doesn't have to be authentication related!).
+
+**Jens:** Integrating the fast-moving world of digital asset custody into the more slow-paced world of traditional finance.
+
+**Dan:** Tell me about your role at Tangany.
+
+**Jens:** I have been a software developer at Tangany for over four years now (wow, time flies...) and have been part of a team building our Partner Portal where customers can manage their access to the APIs provided by Tangany.
+
+**Dan:** How do you use FusionAuth? OAuth? User management? Social sign-on? Something else?
+
+**Jens:** We use FusionAuth for our user management and OAuth 2.0 flows to authenticate our partners in the B2B portal.
+
+> FusionAuth had a more stable developer experience and seemed to be a good balance between reliability, effort to integrate, feature set, and price.
+
+**Dan:** What problems did we solve for you?
+
+**Jens:** We were looking for a reliable authentication service to scale our solution, provide new features, and add additional security mechanisms to the existing system.
+
+**Dan:** How were you solving them before FusionAuth?
+
+**Jens:** We were using another authentication provider.
+
+**Dan:** How are users added to FusionAuth? Is it self-service registration or are you using the APIs to create and register users?
+
+**Jens:** We use the API to add new users to our system.
+
+**Dan:** What security mechanisms or features are on your near-term roadmap?
+
+**Jens:** Step up authentication will enable us to provide a more secure workflow within our applications. We tested this FusionAuth feature during the vendor selection phase.
+
+> Due to regulatory requirements, we required a provider capable of self-hosting.
+
+**Dan:** Why did you choose FusionAuth over the alternatives?
+
+**Jens:** FusionAuth had a more stable developer experience and seemed to be a good balance between reliability, effort to integrate, feature set, and price. Due to regulatory requirements, we required a provider capable of self-hosting.
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
```

Comment:
Should we remove this question and the response?

Reply:
Hi @bradmccarty, I think we should leave it in. I feel like stories are always better if they have real world challenges outlined. They also filed an issue, which I linked to, which highlights our input process.